scripts/qa_crowbarsetup.sh

```diff
@@ -7,11 +7,14 @@ shopt -s extglob
 
 : ${SCRIPTS_DIR:=$(dirname $(readlink -e $BASH_SOURCE))}
 scripts_lib_dir=${SCRIPTS_DIR}/lib
-common_scripts="mkcloud-common.sh"
+common_scripts="mkcloud-common.sh qa_crowbarsetup-help.sh"
 for script in $common_scripts; do
     source ${scripts_lib_dir}/$script
 done
 
+# not being sourced from mkcloud is a feature
+is_onhost && complain 9 "qa_crowbarsetup.sh should not be sourced within mkcloud. Use the common files for shared functions."
```

Comment:
> this can only happen if an old mkcloud accidentally uses a newer `qa_crowbarsetup.sh`

@aspiers 
Thats one part. The other goal is to document this as a feature in order to let PR gatings fail that reintroduce the sourcing later (like the commit that I reverted in this PR).

> This is still cryptic for someone who doesn't understand the issue.

Would it be acceptable to solve this by documenting the current way the mkcloud files are used in a few lines at the top of the file (also describing the common files) and here pointing to that place?